pkg/proto/proto_test.go

```diff
@@ -277,10 +279,67 @@ func TestPeerInfoMarshalJSON(t *testing.T) {
 }
 
 func TestNewPeerInfoFromString(t *testing.T) {
-	rs, err := NewPeerInfoFromString("34.253.153.4:6868")
-	require.NoError(t, err)
-	assert.Equal(t, "34.253.153.4", rs.Addr.String())
-	assert.EqualValues(t, 6868, rs.Port)
+	tests := []struct {
+		in  string
+		out PeerInfo
+		err string
+	}{
+		{"34.253.153.4:6868", PeerInfo{net.IPv4(34, 253, 153, 4), 6868}, ""},
+		{
+			"34.444.153.4:6868",
+			PeerInfo{},
+			"failed to resolve host: failed to resolve host \"34.444.153.4\": lookup 34.444.153.4: no such host",
+		},
+		{
+			"jfhasjdhfkmnn:6868",
+			PeerInfo{},
+			"failed to resolve host: failed to resolve host \"jfhasjdhfkmnn\": lookup jfhasjdhfkmnn: no such host",
```

Comment:
This test fails we different error message, probably because of Docker:
```
--- FAIL: TestNewPeerInfoFromString (0.01s)
    --- FAIL: TestNewPeerInfoFromString/case_2 (0.01s)
        proto_test.go:336: 
            	Error Trace:	/home/runner/work/gowaves/gowaves/pkg/proto/proto_test.go:336
            	Error:      	Error message not equal:
            	            	expected: "failed to resolve host: failed to resolve host \"jfhasjdhfkmnn\": lookup jfhasjdhfkmnn: no such host"
            	            	actual  : "failed to resolve host: failed to resolve host \"jfhasjdhfkmnn\": lookup jfhasjdhfkmnn on 127.0.0.53:53: server misbehaving"
            	Test:       	TestNewPeerInfoFromString/case_2
```

Reply:
Fixed by https://github.com/wavesplatform/gowaves/pull/1551/commits/b3a188c8888f81f06ff2e5473693f5aa3badd9f8